web_src/fomantic/build/semantic.css

```diff
@@ -19897,138 +19897,3 @@ input::selection {
 /*******************************
          Site Overrides
 *******************************/
-/*!
```

Comment:
Did you remove it manually? I think it needs `make fomantic`, and the generated file doesn't have the last new line.